snowpack/src/build/build-import-proxy.ts

```diff
@@ -30,7 +30,7 @@ export function wrapImportMeta({
   env: boolean;
   config: SnowpackConfig;
 }) {
-  if (!code.includes('import.meta')) {
+  if (!/import\s*\.\s*meta/.test(code)) {
```

Comment:
this is a pretty hot code path, can you move the regex creation outside of the function?

Reply:
done